flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -101,7 +101,7 @@ public static void validateConsumerConfiguration(Properties config, List<String>
         }
 
         if (!(config.containsKey(AWSConfigConstants.AWS_REGION)
-                || config.containsKey(ConsumerConfigConstants.AWS_ENDPOINT))) {
+                || config.containsKey(AWSConfigConstants.AWS_ENDPOINT))) {
```

Comment:
Please make sure this change is backwards compatible, remember that customer code can be referencing `ConsumerConfigConstants.AWS_ENDPOINT`

Reply:
It is `ConsumerConfigConstants` extends `AWSConfigConstants` so customer code should not be affected.